ccp/lib/utils/request.py

```diff
@@ -0,0 +1,29 @@
+import requests
+
+
+def request_url(
+        request, params=None, verify_ssl=False, auth=None, headers=None
+):
+    """
+    Queries a specified URL and returns data, if any
+    :param request: The url to send the request to.
+    :type request str
```

Comment:
`:type request: str`

Reply:
applies to all occurrences where `type` is defined.

Reply:
Fixed